### PR TITLE
Disable test, Remove Deprecated autoconfig class

### PR DESCRIPTION
**Pull request by mayur-solace**

### CODE REVIEW

File: solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/pom.xml [hunk 1]
```diff
@@ -102,6 +102,10 @@
             <artifactId>log4j-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
```
Comment (Nephery) [inline solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/pom.xml:108]: Why was this added?

Comment (mayur-solace) [reply solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/pom.xml:108]: We are using org.apache.commons.lang3.exception.ExceptionUtils in binder. 

Earlier this dependency was transitively available. When we removed dependency on java-cfenv we need to explicitly add this dependency.

**Review by Nephery — APPROVED** [review]
